ONSdigital/florence
# Feature/unstub add to collections

### What

Unstubbed and updated collection api methods, added error handling on add dataset and version, and added test coverage to dataset collection controller component

### How to review

- Pull the latest version of Zebedee cmd-develop branch
- Add a dataset and/or a version to a collection via Florence
- Check the dataset and/or version appears in the collection with an `InProgress` state. The dataset/version won't appear on the collections screen, instead check via the api `/zebedee/collectionDetails/{CollectionID}`

### Who can review

Anyone
